mhkit/river/io/usgs.py

```diff
@@ -165,7 +168,12 @@ def request_usgs_data(
 
     # After making the API request and processing the response, write the
     #  response to a cache file
-    handle_caching(hash_params, cache_dir, data=data, clear_cache_file=clear_cache)
+    handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": data, "metadata": None, "write_json": None},
+        clear_cache_file=clear_cache,
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args